client_receiver.py

```diff
@@ -27,16 +24,22 @@ def on_draw():
 
 async def client():
     global state
+    global window
     async with aiohttp.ClientSession() as session:
-        async with session.ws_connect('http://localhost:8080/ws/') as ws:
+        async with session.ws_connect('http://localhost:8080/receiver/') as ws:
             # Waiting for message from server and print them
             async for msg in ws:
-                # Cycle "for" is finished when client disconnect from server
-                if msg.type == aiohttp.WSMsgType.TEXT:
-                    message = msg.data
-                    state_dict = json.loads(message)
-                    state = State.from_dict(state_dict)
-                    print(state_dict)
+                # Cycle "for" is finished when client disconnects from server
+                try:
+                    message = msg.json(loads=json.loads)
+                    state = State.from_dict(message)
+                    window = create_window(state)
```

Comment:
Tohle vůbec nechápu, proč je to vytvoření okna v tom cyklu a ne na začátku kódu.

Reply:
Jak jsem psala: 
> okenko stavajiciho stavu hry, ktery vykresluje client_receiver je male a neda se zvetsit/zmensit. Chtela jsem zaridit, aby se vykreslovalo podobne jako u interface.
Nikde jinde mi to nefungovalo dle ocekavani.